libutils/file_lib.h

```diff
@@ -88,6 +100,7 @@ char *MapName(char *s);
 char *MapNameCopy(const char *s);
 char *MapNameForward(char *s);
 
+Seq *ls(const char *dir, const char *extension);
```

Comment:
I think `ListDir()` or something like that would be a better name now that the function is possibly used more widely.

Reply:
Fixed.